crystalformer/src/von_mises.py

```diff
@@ -104,10 +123,17 @@ def von_mises_logpdf(x, loc, concentration):
     return -(jnp.log(2 * jnp.pi) + jnp.log(jax.scipy.special.i0e(concentration))
               ) + concentration * (jnp.cos((x - loc) % (2 * jnp.pi)) - 1)
 
+def gaussian_logpdf(x, loc, concentration):
+    '''
+    concentration = kappa in von mises distribution = 1/variance
```

Comment:
Docstrings are very serious, it should be writen in a professional way (e.g. the numpy standard: https://numpydoc.readthedocs.io/en/latest/format.html ). If you do not want users to know it, use comments instead.